### PR TITLE
Added a address explorer modal that will be shown when the address of request screen and send confirmation screen is clicked

### PR DESCRIPTION
**Pull request by amanalili**
Task: Tap address to launch blockexplorer

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [x] Tested on iOS Tablet
- [x] Tested on small Android

### CODE REVIEW

File: src/components/common/AddressTextWithBlockExplorerModal.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        yesButtonText: s.strings.string_ok_cap
+      })
+      return (await showModal(modal)) ? Linking.openURL(sprintf(addressExplorer, address)) : null
+    }
+    return Alert.alert(s.strings.modal_addressexplorer_null)
+  }
+  return <TouchableOpacity onPress={addressModal}>{children}</TouchableOpacity>
+}
+
+export default AddressTextWithBlockExplorerModal
```
Comment (paullinator) [inline src/components/common/AddressTextWithBlockExplorerModal.js]: Do not use default exports. Always export/import the exact name of the object

File: src/components/scenes/TransactionDetailsScene.js [hunk 2]
```diff
@@ -443,6 +443,7 @@ export class TransactionDetails extends Component<TransactionDetailsProps, State
   }
 
   render () {
+    console.log(this.props)
```
[the remark this answers is not in the document]
Comment (amanalili) [reply src/components/scenes/TransactionDetailsScene.js]: Noted

File: src/components/common/AddressTextWithBlockExplorerModal.js (hunk 1 above)
Comment (amanalili) [reply src/components/common/AddressTextWithBlockExplorerModal.js]: Noted

**Comment by paullinator** [comment]
Pls resolve conflicts

**Pull request merged**